src/test/java/com/artipie/rpm/misc/FileInDirTest.java

```diff
@@ -46,4 +47,24 @@ void findsFile(@TempDir final Path tmp) throws IOException {
             new IsEqual<>(file)
         );
     }
+
+    @Test
+    void doesNotFindFile(@TempDir final Path tmp) throws IOException {
+        final Path file = tmp.resolve("a_file.txt");
+        Files.write(file, "abc123".getBytes());
```

Comment:
@victornoel You don't need to declare `file`, you could inline it into `Files.write` call